src/logger.cr

```diff
@@ -0,0 +1,72 @@
+class Logger
+  property :level, :progname, :formatter
+
+  UNKNOWN = 5
+  FATAL = 4
+  ERROR = 3
+  WARN = 2
+  INFO = 1
+  DEBUG = 0
+
+  SEV_LABEL = %w(DEBUG INFO WARN ERROR FATAL ANY)
+
+  DEFAULT_FORMATTER = ->(severity : String, datetime : Time, progname : String, message : String) {
+    "#{severity[0]}, [#{datetime} ##{Process.pid}] #{severity.rjust(5)} -- #{progname}: #{message}"
+  }
```

Comment:
One more thing. Maybe the formatter can accept an IO too, and must append to it. That way we can avoid lots of string allocations.